src/views/Pools/index.tsx

```diff
@@ -44,6 +44,7 @@ const Pools: React.FC = () => {
 
   useFetchCakeVault()
   useFetchPublicPoolsData()
+  usePollFarmsData()
```

Comment:
Since the pools need the farms we have to live with fetching all the farms data here as well.

Reply:
Yea makes sense. It's just for prices right? One to think about in future.